.github/workflows/docker-builds.yml

```diff
@@ -31,31 +31,51 @@ permissions: read-all
 
 jobs:
   docker-build:
-    runs-on: [self-hosted, linux.12xlarge]
     environment: ${{ (github.ref == 'refs/heads/main' || startsWith(github.event.ref, 'refs/tags/v')) && 'docker-build' || '' }}
     timeout-minutes: 240
     strategy:
       fail-fast: false
       matrix:
         include:
           - docker-image-name: pytorch-linux-focal-cuda12.1-cudnn8-py3-gcc9
+            runner: linux.12xlarge
```

Comment:
I know repeating the same runner across all these jobs is not elegant. till i get the proper matrix generation working, i'm using it to test the aarch64 workflow.

Reply:
updated the matrix generation logic in the next version.